File: lecture010/fibo.cpp
#include<iostream>

using namespace std ;
int fibo(int n){
    if(n<=1)
    return n ;

    int recans = fibo(n-1) + fibo(n-2) ;
    return recans;
}
int main(int args , char** argv){
    int a;
    cout<<"enter the position in fibo series "<<endl;
    cin>>a;
    cout<<"number at position " <<a<<" is "<<fibo(a)<<endl;
}

File: lecture010/fibo.java
import java.util.Scanner;

public class fibo {
    public static Scanner scn = new Scanner(System.in);

    public static void main(String[] args) {
        System.out.println("enter positon of a num ");
        int a = scn.nextInt();
        System.out.println("num at position " + a + " is " + fibo_(a));
    }

    public static int fibo_(int a) {

        if (a <= 1)
            return a;

        int recAns = fibo_(a - 1) + fibo_(a - 2);
        return recAns;

    }
}